oauth2client/service_account.py

```diff
@@ -27,14 +27,14 @@
 from oauth2client import GOOGLE_TOKEN_URI
 from oauth2client import util
 from oauth2client._helpers import _from_bytes
-from oauth2client.client import _apply_user_agent
-from oauth2client.client import _initialize_headers
 from oauth2client.client import _UTCNOW
 from oauth2client.client import AccessTokenInfo
 from oauth2client.client import AssertionCredentials
-from oauth2client.client import clean_headers
 from oauth2client.client import EXPIRY_FORMAT
 from oauth2client.client import SERVICE_ACCOUNT
+from oauth2client.transport import _apply_user_agent
```

Comment:
Have we today any better options for avoiding violations of "don't import private members of other modules" than we did when the left-side-of-the-diff code was written?